Tests/SwiftStdlibTests/ArrayExtensionsTests.swift

```diff
@@ -91,4 +95,20 @@ final class ArrayExtensionsTests: XCTestCase {
         let arrayWithoutDuplicatesNHashablePrepared = [Person(name: "Wade", age: 20, location: Location(city: "London")), Person(name: "James", age: 32), Person(name: "James", age: 72, location: Location(city: "Moscow")), Person(name: "Wade", age: 22, location: Location(city: "Prague"))]
         XCTAssertEqual(arrayWithoutDuplicatesNHashable, arrayWithoutDuplicatesNHashablePrepared)
     }
+    
+    func testDistance() {
+        #if canImport(CoreLocation)
+        let locations = [
+            CLLocation(latitude: 41.0, longitude: -111.0),
+            CLLocation(latitude: 41.0, longitude: -112.0),
+            CLLocation(latitude: 41.0, longitude: -113.0)
+        ]
+
+        XCTAssertEqual(locations.distance(unit: .meter), 168270.37022814364)
```

Comment:
We don't have to do exactly comparations here, we can use a `XCTAssertEqual(FloatingPoint, FloatingPoint, accuracy: FloatingPoint)` e.g. `XCTAssertEqual(locations.distance(unit: .kilometer), 168.27, accuracy: 0.01)`